gradle/update-gh-pages.gradle

```diff
@@ -157,6 +149,20 @@ updateGitHubPages.doLast {
 
     final def GH_PAGES_BRANCH = "gh-pages"
 
+    final String repoSlug = System.getenv('TRAVIS_REPO_SLUG')
+    if (repoSlug == null || repoSlug.isEmpty()) {
+        throw new GradleException('`TRAVIS_REPO_SLUG` environmental variable is not set.')
+    }
+
+    /**
+     * The GitHub URL to the project repository.
+     *
+     * <p>The host name is {@code github.com-publish}, though resolved to {@code github.com} by SSH on
+     * Travis due to the SSH configuration. This is used to make it possible to have multiple SSH
+     * private keys on Travis (since there is one already on Travis).
```

Comment:
Please clarify.

No one ever will get the original issue from the `(since there is one already on Travis)`. I would describe the original issue first. Then explain why `github.com-publish` is used.